astropy/io/ascii/cparser.pyx

```diff
@@ -6,6 +6,7 @@ import os
 import math
 import multiprocessing
 import mmap
+import queue as Queue
```

Comment:
nitpicking, but there is reason to call it `Queue` any more?

Reply:
Yes, `queue` is used as variable name in the code. I could rename these but it seems easier to keep the import like this.

Reply:
ah, ok, fair enough, I didn't look into the code.